proto/opamp.proto

```diff
@@ -180,6 +188,14 @@ message ServerToAgent {
     // will be performed.
     // Status: [Beta]
     ServerToAgentCommand command = 9;
+
+    // A message indicating custom capabilities supported by the Server.
+    // Status: [Development]
+    CustomCapabilities custom_capabilities = 10;
```

Comment:
Is it possible to omit this? With a long list of capabilities this can become significant payload size-wise.

Reply:
The intent is that this is only sent when it changes, but we could also add another flag for `ReportCapabilities` for both the ServerToAgent and AgentToServer messages.

Reply:
>The intent is that this is only sent when it changes

This is fine, let's make sure it is documented. I don't see it mentioned in the spec.

Reply:
I added another sentence to the CustomCapabilities section which should clarify this.

> CustomCapabilities

> The Agent and Server should both use this message to signal that they support specific custom capabilities. It is supported on ServerToAgent and AgentToServer. This message should only be sent when the list of supported custom capabilities changes. When this message is sent, the list of supported custom capabilities should be updated to match this list. If this message is never sent, it is assumed that no custom capabilities are
supported.